### PR TITLE
Printable PDF Generation

### PR DESCRIPTION
**Pull request by drigodwin**
Produces a second, more printer friendly version of the PDF documentation. Uses an additional style sheet to remove colours and formatting unsuitable for printing.

### CODE REVIEW

File: _build/build.sh [hunk 1]
```diff
@@ -192,9 +192,11 @@ function build_pdf() {
   
   if [ "$SKIP_PDF_MANUAL" != "true" ]; then
     _build/buildPDF.sh "_site/zoneMergeManual.html" "_pdf/UserManual.pdf" 2>&1 | tee $PDF_MANUAL_LOG
+    _build/buildPDF.sh "_site/zoneMergeManual.html" "_pdf/UserManual_printable.pdf" "./style/css/printable.css" 2>&1 | tee $PDF_MANUAL_LOG
```
Comment (ahgittin) [inline _build/build.sh]: not related to this PR but is there a reason this isn't `build-pdf.sh` consistent with the other commands in the dir?

File: _build/buildPDF.sh [hunk 2]
```diff
@@ -18,7 +18,11 @@ if ! command -v wkhtmltopdf >/dev/null 2>&1; then
 fi
 	
 # run the PDF build with wkhtmltopdf
-wkhtmltopdf --page-size Letter --margin-top 0.75in --margin-right 0.75in --margin-bottom 0.75in --margin-left 0.75in --encoding UTF-8 $1 $2
+if [ -z "$3" ] ; then
+	wkhtmltopdf --page-size Letter --margin-top 0.75in --margin-right 0.75in --margin-bottom 0.75in --margin-left 0.75in --encoding UTF-8 $1 $2
+else
+	wkhtmltopdf --user-style-sheet $3 --page-size Letter --margin-top 0.75in --margin-right 0.75in --margin-bottom 0.75in --margin-left 0.75in --encoding UTF-8 $1 $2
```
Comment (ahgittin) [inline _build/buildPDF.sh]: could revert from `if/then` to a single line using `${parameter:+word}`

    wkhtmltopdf ${3:+--user-style-sheet $3} --page-size Letter ...

(i can never remember this bash magic, i always do `man bash` then search for `##` to find parameter expansion...)

File: style/css/printable.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/*
+ * This is a CSS included in the printable version of docs, it's used to mask out large images etc
+ */
```
Comment (ahgittin) [inline style/css/printable.css]: not sure this does anything to large images yet but makes sense that would be the intention (no need to change if it is)

**Comment by ahgittin** [comment]
very minor, then good to merge

**Comment by drigodwin** [comment]
Thankyou for the comments @ahgittin !

**Comment by aledsage** [comment]
+1; LGTM - merging now.

**Pull request merged**